xwiki-pro-commons-pickers/xwiki-pro-commons-pickers-api/src/main/resources/templates/pickers_macros.vm

```diff
@@ -0,0 +1,32 @@
+## ---------------------------------------------------------------------------
+## See the NOTICE file distributed with this work for additional
+## information regarding copyright ownership.
+##
+## This is free software; you can redistribute it and/or modify it
+## under the terms of the GNU Lesser General Public License as
+## published by the Free Software Foundation; either version 2.1 of
+## the License, or (at your option) any later version.
+##
+## This software is distributed in the hope that it will be useful,
+## but WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+## Lesser General Public License for more details.
+##
+## You should have received a copy of the GNU Lesser General Public
+## License along with this software; if not, write to the Free
+## Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+## 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+## ---------------------------------------------------------------------------
+#macro (tagPicker_import)
+  #picker_import
+  #set ($discard = $xwiki.jsx.use('pickers.tagpicker'))
```

Comment:
```suggestion
  #set ($discard = $xwiki.jsx.use('XWiki Pro Commons.Pickers.TagPicker'))
```
I think it would be useful to have all the code in the same space. This and the future additions.